tests/ProportionalDerivativeControllerTest.cpp

```diff
@@ -0,0 +1,86 @@
+/**
+ * @file ProportionalDerivativeControllerTest.cpp
+ * @author Giulio Romualdi
+ * @copyright Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2020
+ */
+
+// Catch2
+#include <catch2/catch.hpp>
+
+#include <LieGroupControllers/ProportionalDerivativeController.h>
+using namespace LieGroupControllers;
+
+TEST_CASE("Proportional Derivative Controller [SO(3)]")
+{
+    manif::SO3d desiredState, state;
+    desiredState.setRandom();
+    state.setRandom();
+    manif::SO3d::Tangent stateDerivative = Eigen::Vector3d::Zero();
+    const manif::SO3d::Tangent desiredStateDerivative = Eigen::Vector3d::Zero();
+    const auto feedForward = Eigen::Vector3d::Zero();
+
+    constexpr double dT = 0.01;
+    constexpr double kp = 10;
+    constexpr double kd = 2 * std::sqrt(kp);
+    constexpr std::size_t numberOfIteration = 1e3;
+
+    ProportionalDerivativeControllerSO3 controller;
+    controller.setGain({kp, kd});
+    controller.setDesiredState({desiredState, desiredStateDerivative});
+
+    controller.setFeedForward(feedForward);
+
+    for (std::size_t i = 0; i < numberOfIteration; i++)
+    {
+        controller.setState({state, stateDerivative});
+        controller.computeControlLaw();
+        auto controlOutput = controller.getControl();
+        manif::SO3d::Tangent stateDerivativeDT = stateDerivative.coeffs() * dT;
+        state = stateDerivativeDT + state;
+
+        manif::SO3d::Tangent controlOutputDT = controlOutput.coeffs() * dT;
+        stateDerivative += controlOutput * dT;
```

Comment:
It would be really helpful if you documented this overloaded operator or pointed towards the existing manif documentation, to understand what is being done under the hood.

Reply:
-  [`operator *()`](https://github.com/artivis/manif/blob/6d07bc65cc5b25c49f1231021be5e61132e5f777/include/manif/impl/tangent_base.h#L316-L318)
- [`operator +=()`](https://github.com/artivis/manif/blob/6d07bc65cc5b25c49f1231021be5e61132e5f777/include/manif/impl/tangent_base.h#L300-L310)

Actually I noticed that 
```cpp
manif::SO3::Tangent a, b, c;
double number = 3;
a.setRandom();
b = a * number; // <--------- Works
c = number * a; // <-------- Fails
```
and the error is 
```
error: no match for ‘operator*’ (operand types are ‘const double’ and ‘manif::SO3Tangent<double>’)
```

Reply:
FYI [this PR](https://github.com/artivis/manif/pull/153) adds the missing operator.